### PR TITLE
Adding flask route for malaria model and required frontend pages for the same

### PR DESCRIPTION
**Pull request by ayush-raj8**
Added required changes in app.py , developed frontend pages. modified index.html to add malaria link, modified requirements.txt to add new libraries

##Adding flask route for malaria model and required frontend pages for the same

Adding flask route for malaria model and required frontend pages for the same

Closes: #177 

#### Describe the changes you've made

Added required changes in app.py , developed frontend pages. modified index.html to add malaria link, modified requirements.txt to add new libraries for image processing
## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.

https://user-images.githubusercontent.com/52033949/114519670-45646b80-9c5e-11eb-9981-2d8974dbefcc.mp4

### CODE REVIEW

**Review by shubhigupta991 — APPROVED** [review]
Looks good to me❤️

**Comment by ayush-raj8** [comment]
please add appropriate level to this too.

File: app.py [hunk 1]
```diff
@@ -196,5 +200,32 @@ def predict_stroke():
...
+def malariapredict():
+    if request.method == 'POST':
+        try:
+            if 'image' in request.files:
+                img = Image.open(request.files['image'])
+                img = img.resize((50,50))
+                img = np.asarray(img)
+                img = img.reshape((1,50,50,3))
+                img = img.astype(np.float64)
+                model = load_model("Models/malaria-model.h5")
```
Comment (kritikaparmar-programmer) [inline app.py:220]: why you have loaded the model two times?

**Review by kritikaparmar-programmer — CHANGES_REQUESTED** [review]

**Comment by kritikaparmar-programmer** [comment]
Hey, @ayush-raj8 please resolve the conflicts.

**Review by ayush-raj8 — COMMENTED** [review]
> Hey, @ayush-raj8 please resolve the conflicts.

Conflict has been resolved.  Removed the redundant line , check apps.py
image

**Comment by ayush-raj8** [comment]
@kritikaparmar-programmer  @utkarsh0702  please assign level and update it.

**Pull request merged**